ubuntu.json

```diff
@@ -50,22 +50,15 @@
     {
       "boot_command": [
         "{{ user `boot_command_prefix` }}",
-        "/install/vmlinuz ",
-        "noapic ",
-        "auto ",
+        "/install/vmlinuz noapic<wait> ",
         "initrd=/install/initrd.gz ",
-        "file=/floppy/{{ user `preseed` }} ",
-        "debian-installer={{ user `locale` }} ",
-        "locale={{ user `locale` }} ",
-        "kbd-chooser/method=us ",
+        "file=/floppy/{{ user `preseed` }}<wait> ",
+        "debian-installer={{ user `locale` }} auto locale={{ user `locale` }} kbd-chooser/method=us ",
         "hostname={{ user `hostname` }} ",
         "grub-installer/bootdev=/dev/sda<wait> ",
-        "fb=false ",
-        "debconf/frontend=noninteractive ",
-        "keyboard-configuration/modelcode=SKIP ",
-        "keyboard-configuration/layout=USA ",
-        "keyboard-configuration/variant=USA ",
-        "console-setup/ask_detect=false ",
```

Comment:
did you joint multiple lines into a single line by intent here? just curious, makes it slightly harder to recognize the actual diff

Reply:
Yes, it's essentially the same boot command, I just copied the boot command from vmware with the additional waits.
The reason for the change is that apparently the VM would not receive all keystrokes during boot and fail since the boot command was not valid. With the additional waits it now works, at least on my machine.